Library/Formula/pillar.rb

```diff
@@ -0,0 +1,26 @@
+class Pillar < Formula
+  homepage "https://github.com/comeara/pillar"
+  url "https://github.com/comeara/pillar/archive/v2.0.1.tar.gz"
+  sha256 "d5db7fbf956dae05794ec7b26c1de70aaab1d21fed774d95d16a241e9dd3952e"
+
+  depends_on :java
+  depends_on "sbt" => :build
+
+  def install
+    system "sbt", "assembly"
+
+    inreplace "src/main/bash/pillar" do |s|
+      s.gsub! "/usr/java/default", "`/usr/libexec/java_home`"
+      s.gsub! "${PILLAR_ROOT}/lib/pillar.jar", "#{libexec}/pillar-assembly-#{version}.jar"
+      s.gsub! "${PILLAR_ROOT}/conf", "#{etc}/pillar-log4j.properties"
+    end
+
+    bin.install "src/main/bash/pillar"
+    etc.install "src/main/resources/pillar-log4j.properties"
+    libexec.install "target/scala-2.10/pillar-assembly-#{version}.jar"
+  end
+
+  test do
+    assert_match /Missing parameter/, shell_output("#{bin}/pillar 2>&1", 1)
```

Comment:
As said above, that would require a running Cassandra instance. Also you would test only

1. that the application is properly installed & executable - this is already ensured by the current test
2. that running ```pillar initialize myapp``` has a return value of ```0``` - otherwise you would also have to check if keyspaces within Cassandra got manipulated.

Pillar already contains a test suite, which should get run during release creation, which actually verifies it is working properly. Therefore such a test wouldn't - IMHO - really add additional value while forcing a local Cassandra installation on everyone (which is seldom wanted) and making the test much more compilcated (install cassandra, configure a standalone instance, start it, wait until it is ready, ...).